search/includes/functions/ep-get-query-log.php

```diff
@@ -0,0 +1,12 @@
+<?php
+require_once __DIR__ . '../../../elasticpress/elasticpress.php';
```

Comment:
Aside: these traversals are going to bite us at some point and we need to stop doing them. In a future PR probably best to define a constant called `VIP_SEARCH__DIR__` and switch to relative paths from there.

/cc @nickdaugherty @chrean @hanifn